src/ripple/app/misc/Validations.cpp

```diff
@@ -93,6 +93,10 @@ class ValidationsImp : public Validations
 
         auto pubKey = app_.validators ().getTrustedKey (signer);
 
+        // Do not process or forward partial validations.
```

Comment:
Just for my own education, what are the consequences of not forwarding partial validations?  My understanding is that part of the role of a partial validation is that it lets other folks know that validator is still on the job, although it is not contributing during this round.  Will not forwarding the partial cause some observers to remove this validator?